src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -61,6 +61,19 @@ const AudioInputVFControl: React.FC<Props> = ({
     };
   }, []);
 
+  useEffect(() => {
+    // When user turns on Amazon Voice Focus and the default device changes in Chrome,
+    // we will wrap Amazon Voice Focus to the default device.
+    const wrapVoiceFocus = async () => {
+      if (isVoiceFocusOn) {
+        const currentDevice = await addVoiceFocus(selectedDevice);
+        await meetingManager.selectAudioInputDevice(currentDevice);
+      }
+    }
+    
+    wrapVoiceFocus();
+  }, [devices[0].label]);
```

Comment:
Why not watch `devices` altogether?

Or why not specifically watch the group ID of the default device?

Reply:
If the device changes but Voice Focus isn't on, should you do something?

Reply:
So how come we won't be racing against `AudioInputProvider`?

(Why are we adding `440` etc. to the device list?!)

What problem exactly are you trying to solve here?

Reply:
>Why are we adding 440 etc. to the device list?!

I don't know...

>So how come we won't be racing against AudioInputProvider?

`AudioInputProvider` will first call `audioVideo?.chooseAudioInputDevice("default")` and then update the `devices`. After `devices` gets updated, this useEffect() will be triggered.

>What problem exactly are you trying to solve here?

When user selects "default" device as audio input device in Chrome (no Safari, no FF), and enables Amazon Voice Focus. Then a new device (AirPods) is plugged in, the "default" device will automatically changes to AirPods by `audioVideo?.chooseAudioInputDevice(selectedInputRef.current);`. However, it is just a normal audio device without VF. So we need to add VF to this "default" device. But we only `addVoiceFocus` when user click a different device or turn on VF. So need to add a new `useEffect` to keep an eye on the change of default device, then we can add VF to it in time.

Reply:
> Why are we adding 440 etc. to the device list?


I think this was added since the beginning and probably just followed the JS SDK meeting demo. We have been called out on that https://github.com/aws/amazon-chime-sdk-component-library-react/issues/629

Reply:
>the React app will leak a burst of noise for a few hundred milliseconds, right?

Yes, you are right...
Will try the approaches you mentioned